### PR TITLE
Render synthetic nodes in `jj log` by default

### PR DESCRIPTION
**Pull request by martinvonz**
# Checklist

If applicable:
- [x] I have updated `CHANGELOG.md`
- [x] I have updated the documentation (README.md, docs/, demos/)
- [x] I have updated the config schema (cli/src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

**Comment by thoughtpolice** [comment]
Do we want to do this while `DOTTED CIRCLE` still renders so badly in some terminals? I suspect we'll get complaints immediately. But I guess, probably any choice might end up the same way, because fonts and terminals and unicode are all strange beasts...

**Comment by martinvonz** [comment]
> Do we want to do this while `DOTTED CIRCLE` still renders so badly in some terminals? I suspect we'll get complaints immediately. But I guess, probably any choice might end up the same way, because fonts and terminals and unicode are all strange beasts...

Good question. I've asked for clarification in https://github.com/martinvonz/jj/discussions/3238#discussioncomment-8723499

**Comment by martinvonz** [comment]
Turns out the harpoons work any better (thanks for checking, @amiryal) :( So what do you think, @thoughtpolice? I'm leaning towards enabling the synthetic nodes despite the ugly rendering in some terminals.

File: cli/src/commands/log.rs [hunk 1]
```diff
@@ -27,14 +27,18 @@ use crate::diff_util::{self, DiffFormatArgs};
 use crate::graphlog::{get_graphlog, Edge};
 use crate::ui::Ui;
 
-/// Show commit history
+/// Show revision history
+///
+/// Renders a graphical view of the project's history with, ordered with
```
Comment (yuja) [inline cli/src/commands/log.rs:32]: nit: redundant "with, " ?

**Review by yuja — APPROVED** [review]

**Comment by martinvonz** [comment]
Ping, @thoughtpolice. If I don't hear from you, I'll merge this tomorrow.

**Review by thoughtpolice — APPROVED** [review]

**Pull request merged**